modules/vaos/app/docs/vaos/v2/vaos_v2.yaml

```diff
@@ -93,6 +93,56 @@ paths:
           $ref: "#/components/responses/ServiceUnavailable"
         "504":
           $ref: "#/components/responses/GatewayTimeout"
+  "/patients/{icn}/appointments/{appointment-id}":
```

Comment:
I believe that ICN will be populated automatically in the service layer, we shouldn't need to pass it explicitly as a parameter.